examples/flutter_gallery/lib/demo/expansion_panels_demo.dart

```diff
@@ -0,0 +1,335 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+enum _Location {
+  Barbados,
+  Bahamas,
+  Bermuda
+}
+
+const TextStyle text54 = const TextStyle(
+  color: Colors.black54,
+  fontSize: 15.0
+);
+const TextStyle text87 = const TextStyle(
+  color: Colors.black87,
+  fontSize: 15.0
+);
+
+typedef Widget DemoItemBodyBuilder(DemoItem<dynamic> item);
+typedef String ValueToString<T>(T value);
+
+class DualHeaderWithHint extends StatelessWidget {
+  DualHeaderWithHint({
+    this.name,
+    this.value,
+    this.hint,
+    this.showHint
+  });
+
+  final String name;
+  final String value;
+  final String hint;
+  final bool showHint;
+
+  Widget _crossFade(Widget first, Widget second, bool isExpanded) {
+    return new AnimatedCrossFade(
+      firstChild: first,
+      secondChild: second,
+      firstCurve: new Interval(0.0, 0.6, curve: Curves.fastOutSlowIn),
+      secondCurve: new Interval(0.4, 1.0, curve: Curves.fastOutSlowIn),
+      sizeCurve: Curves.fastOutSlowIn,
+      crossFadeState: isExpanded ? CrossFadeState.showSecond : CrossFadeState.showFirst,
+      duration: const Duration(milliseconds: 200),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return new Row(
+      children: <Widget>[
+        new Flexible(
+          flex: 2,
+          child: new Container(
+            margin: const EdgeInsets.only(left: 24.0),
+            child: new Text(name, style: text87)
+          )
+        ),
+        new Flexible(
+          flex: 3,
+          child: new Container(
+            margin: const EdgeInsets.only(left: 24.0),
+            child: _crossFade(
```

Comment:
We don't usually call a function for a case like this, just inline the AnimatedCrossFade